src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -134,7 +134,7 @@ def get_text_responses(
             A list of responses to the prompts. The responses can be None if the
             evaluation fails.
         """
```

Comment:
Shall we add a warning to tell that the default model has changed? People might get surprised if the behavior of `OpenAIEvalClient()` suddenly changes.

Reply:
Added warnings: [8c1319f](https://github.com/citadel-ai/langcheck/pull/186/commits/8c1319f335ae048dac08b88c8d3fe6016a3d0d13)